src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -15,19 +15,23 @@
 
 (defn new-account
   []
-  [quo/action-drawer
-   [[{:icon                :i/add
-      :accessibility-label :start-a-new-chat
-      :label               (i18n/label :t/add-account)
-      :sub-label           (i18n/label :t/add-account-description)
-      :on-press            #(rf/dispatch [:navigate-to :screen/wallet.create-account])}
-     (when (ff/enabled? ::ff/wallet.add-watched-address)
-       {:icon                :i/reveal
-        :accessibility-label :add-a-contact
-        :label               (i18n/label :t/add-address-to-watch)
-        :sub-label           (i18n/label :t/add-address-to-watch-description)
-        :on-press            #(rf/dispatch [:navigate-to :screen/wallet.add-address-to-watch])
-        :add-divider?        true})]]])
+  (let [keycard? (rf/sub [:keycard/keycard-profile?])]
+    [quo/action-drawer
+     [[{:icon                :i/add
+        :accessibility-label :start-a-new-chat
+        :label               (i18n/label :t/add-account)
+        :sub-label           (i18n/label :t/add-account-description)
+        :on-press            (fn []
+                               (if keycard?
+                                 (rf/dispatch [:keycard/feature-unavailable-show])
```

Comment:
Can we change this line to be something like this?
```
(rf/dispatch [:keycard/feature-unavailable-show {:feature-name :chat.new-chat}])
```
This will help us with metrics that we track for which unavailable feature are people discovering.

Reply:
Good catch @seanstrom 👍🏼

Reply:
done